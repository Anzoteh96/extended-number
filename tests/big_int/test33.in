3511808816873195458767643204778416594205842357167860627223849632500915629070573366412430091540557463
712859440603457315129089387808054526864520214034840229992785679148136188800739457323445185538942284
