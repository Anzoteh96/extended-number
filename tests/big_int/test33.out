Enter two numbers n and m:
n is 3511808816873195458767643204778416594205842357167860627223849632500915629070573366412430091540557463
m is 712859440603457315129089387808054526864520214034840229992785679148136188800739457323445185538942284
n + m is 4224668257476652773896732592586471121070362571202700857216635311649051817871312823735875277079499747
n - m is 2798949376269738143638553816970362067341322143133020397231063953352779440269833909088984906001615179
n * m is 2503426068702515385624607702365351625126481780641919017589820454548274462850536739667006612695918100410364642535032903260042296933145079642095382194287254146759998292350164941704094148362772742465492
n / m is 4
n % m is 660371054459366198251285653546198486747761501028499707252706915908370873867615537118649349384788327
